docs/contributing/backend/api/pipelines.mdx

```diff
@@ -6,48 +6,47 @@ block dependencies."
 
 ## Pipeline object
 
-
 ```json
 {
   "blocks": [
-    {
```

Comment:
You don't need to remove the indent here

Reply:
I used VS code to convert from four spaces to two and this was a byproduct. I'll fix